getting_started/workflow/export/exporting_for_android.rst

```diff
@@ -32,13 +38,17 @@ the JDK can be used for this purpose:
 
     keytool -keyalg RSA -genkeypair -alias androiddebugkey -keypass android -keystore debug.keystore -storepass android -dname "CN=Android Debug,O=Android,C=US" -validity 9999
 
+- this will create a ```debug.keystore-``` file in your current directory.  you should move it to ```%userprofile%/.android/```
```

Comment:
```suggestion
- This will create a ``debug.keystore`` file in your current directory. You should move it to ``%USERPROFILE%\.android\``.
```

Also, is it actually necessary to move this file? As far as I know, it can be located anywhere.

Maybe Godot could be modified to use a default keystore path, which would remove the need to configure the debug keystore path in the Editor Settings.

Reply:
you are right, the ``debug.keystore`` can be anywhere, the user just needs to remember the location for a further step.   I don't know how to apply your edit, so maybe you can add that "remember the location" details.